integration-tests/scaffolding/extrinsicHelpers.ts

```diff
@@ -24,81 +29,76 @@ type AddProviderPayload = { authorizedMsaId?: any; schemaIds?: any; expiration?:
  * up in the test.
  */
 
+
+/** Generic wrapper **/
+export async function signAndSend<T extends ISubmittableResult>(
+    f: () => SubmittableExtrinsic<"rxjs", T>,
+    keys: KeyringPair,
+    customErrorHandler: (err: RpcError, caught: Observable<T>) => Observable<T> = (err, caught) => caught): Promise<EventMap> {
+    return firstValueFrom(f().signAndSend(keys).pipe(
+        filter(({ status }) => status.isInBlock || status.isFinalized),
+        catchError((err: RpcError, caught) => {
+            console.log(`code: ${err.code}, name: ${err.name}, message: ${err.message}`);
```

Comment:
This log feels wrong as we get it even when we are expecting it. Perhaps something else we might want on a flag?